.github/ISSUE_TEMPLATE/2_new-hub-provide-info.yml

```diff
@@ -1,6 +1,6 @@
-name: 💻  New Hub
-description: Creating a new hub for 2i2c to operate
-title: "[New Hub] {{ HUB NAME }}"
+name: 💻  New Hub - Information Provider
```

Comment:
I feel there is a better name for this than `provider`, but I can't seem to find it?